pkg/webui/lib/selectors/id.js

```diff
@@ -16,7 +16,7 @@ import getByPath from '../get-by-path'
 
 export const getApplicationId = function (application = {}) {
   return getByPath(application, 'application_id')
-    || getByPath(application, 'application_ids.application_id')
+    || getByPath(application, 'ids.application_id')
```

Comment:
this breaks the events component.
my suggestion would be to add an additional case:
```suggestion
    || (getByPath(application, 'application_ids.application_id')
    || (getByPath(application, ''ids.application_id'))
```

Reply:
This also causes the issue we discussed in slack:
```
checkPropTypes.js:19 Warning: Failed prop type: The prop `emitter` is marked as required in `Event`, but its value is `undefined`.
    in Event (created by DefaultEvent)
    in DefaultEvent (created by List)
    in li (created by ListItem)
    in ListItem (created by List)
    in ol (created by List)
    in div (created by List)
    in List (created by EventsWidget)
    in aside (created by EventsWidget)
    in EventsWidget (created by EventsSubscription)
    in EventsSubscription (created by Connect(EventsSubscription))
```

Reply:
Ahh indeed. Was looking whether it was used elsewhere, but couldn't find it. Will add a case then.